src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -116,6 +141,14 @@ else if (o instanceof Slot)
 				case REMOVE_ALL:
 					assert false;
 			}
+			if (durability &&
+				itemStack != null &&
+				mode != ChangeMode.RESET &&
+				mode != ChangeMode.DELETE) {
+				Bukkit.broadcastMessage(changeValue + "");
```

Comment:
Could you explain what this broadcast is for please?

Reply:
oh oops, it was there for debugging, forgot to remove it 😅